compiler/x/env/OMRCPU.cpp

```diff
@@ -144,5 +150,8 @@ OMR::X86::CPU::getX86OSSupportsSSE2(TR::Compilation *comp)
 bool
 OMR::X86::CPU::getX86SupportsTM(TR::Compilation *comp)
    {
-   return false;
+   uint32_t flags8 = self()->getX86ProcessorFeatureFlags8(comp);
```

Comment:
Hi Andrew, I added another commit in my PR. I added another processor info for Broadwell and we assume only the version after broadwell supporting TM. (Even though a small number of Broadwell also have such problem, ignore it for now and I leave a TODO there. )